docs/docusaurus/docs/snippets/checkpoints_and_actions.py

```diff
@@ -1,469 +0,0 @@
-import os
-
-import great_expectations as gx
-from great_expectations.core.expectation_validation_result import (
-    ExpectationSuiteValidationResult,
```

Comment:
Checkpoints and actions follow very different patterns in V1 so deleting this file